lib/intersection_validator.rb

```diff
@@ -0,0 +1 @@
+IntersectionValidator = Class.new(ActiveModelValidatesIntersectionOf::Validator)
```

Comment:
I'm just follow the Ruby Styleguide , please feel free to change it after merging :)

> Prefer a single-line format for class definitions with no body.

https://github.com/bbatsov/ruby-style-guide#single-line-classes